{"uid":"94660e53b6b8f75a","name":"test_smoke","fullName":"tests.integration.test_smoke#test_smoke","historyId":"78ffebf457376473b3476ee0f7a454c9","time":{"start":1750555441698,"stop":1750555442849,"duration":1151},"description":"Deploy bundle with app and release as a product.","descriptionHtml":"<p>Deploy bundle with app and release as a product.</p>\n","status":"broken","statusMessage":"juju.errors.JujuError: resolving ch:grafana-agent : selecting releases: charm or bundle not found for channel \"latest/edge\", base \"amd64/ubuntu/22.04\"\navailable releases are:\n  channel \"1/stable\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"1/candidate\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"1/beta\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"1/edge\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"latest/candidate\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"latest/beta\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"latest/edge\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"2/edge\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/tests/integration/test_smoke.py\", line 68, in test_smoke\n    await ops_test.model.deploy(\"./releases/latest/mysql-bundle.yaml\")\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/juju/model.py\", line 1764, in deploy\n    await handler.execute_plan()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/juju/bundle.py\", line 408, in execute_plan\n    await self._resolve_charms()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/juju/bundle.py\", line 394, in _resolve_charms\n    charm_url, charm_origin = await self.model._resolve_charm(charm_url, origin)\n                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/juju/model.py\", line 1919, in _resolve_charm\n    raise JujuError(f'resolving {url} : {result.error.message}')\njuju.errors.JujuError: resolving ch:grafana-agent : selecting releases: charm or bundle not found for channel \"latest/edge\", base \"amd64/ubuntu/22.04\"\navailable releases are:\n  channel \"1/stable\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"1/candidate\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"1/beta\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"1/edge\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"latest/candidate\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"latest/beta\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"latest/edge\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"2/edge\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1750555441541,"stop":1750555441542,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1750555441542,"stop":1750555441697,"duration":155},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1750555441542,"stop":1750555441542,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1750555441697,"stop":1750555441697,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Deploy bundle with app and release as a product.","status":"broken","statusMessage":"juju.errors.JujuError: resolving ch:grafana-agent : selecting releases: charm or bundle not found for channel \"latest/edge\", base \"amd64/ubuntu/22.04\"\navailable releases are:\n  channel \"1/stable\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"1/candidate\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"1/beta\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"1/edge\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"latest/candidate\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"latest/beta\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"latest/edge\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"2/edge\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/tests/integration/test_smoke.py\", line 68, in test_smoke\n    await ops_test.model.deploy(\"./releases/latest/mysql-bundle.yaml\")\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/juju/model.py\", line 1764, in deploy\n    await handler.execute_plan()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/juju/bundle.py\", line 408, in execute_plan\n    await self._resolve_charms()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/juju/bundle.py\", line 394, in _resolve_charms\n    charm_url, charm_origin = await self.model._resolve_charm(charm_url, origin)\n                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.12/site-packages/juju/model.py\", line 1919, in _resolve_charm\n    raise JujuError(f'resolving {url} : {result.error.message}')\njuju.errors.JujuError: resolving ch:grafana-agent : selecting releases: charm or bundle not found for channel \"latest/edge\", base \"amd64/ubuntu/22.04\"\navailable releases are:\n  channel \"1/stable\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"1/candidate\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"1/beta\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"1/edge\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"latest/candidate\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"latest/beta\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"latest/edge\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"2/edge\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04","steps":[],"attachments":[{"uid":"fef03a46a9fa715a","name":"log","source":"fef03a46a9fa715a.txt","type":"text/plain","size":591}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1750555442854,"stop":1750555442854,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1750555442854,"stop":1750555443117,"duration":263},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1750555443118,"stop":1750555443118,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1750555443118,"stop":1750555443118,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1750555443118,"stop":1750555443118,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_smoke"},{"name":"host","value":"pkrvmxyh4eaekms"},{"name":"thread","value":"21409-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_smoke"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"8721e2cc283c989e","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[]}],"history":{"statistic":{"failed":59,"broken":16,"skipped":0,"passed":233,"unknown":12,"total":320},"items":[{"uid":"5a86d5d42ef66bab","reportUrl":"../1815//#testresult/5a86d5d42ef66bab","status":"broken","statusDetails":"juju.errors.JujuError: resolving ch:grafana-agent : selecting releases: charm or bundle not found for channel \"latest/edge\", base \"amd64/ubuntu/22.04\"\navailable releases are:\n  channel \"latest/edge\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"2/edge\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"1/stable\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"1/candidate\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"1/beta\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"1/edge\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"latest/candidate\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04\n  channel \"latest/beta\": available bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04","time":{"start":1750469015943,"stop":1750469017440,"duration":1497}},{"uid":"8f8b24b2991f26e3","reportUrl":"../1814//#testresult/8f8b24b2991f26e3","status":"passed","time":{"start":1750382660006,"stop":1750383750514,"duration":1090508}},{"uid":"c1938cb06e3c0ce5","reportUrl":"../1813//#testresult/c1938cb06e3c0ce5","status":"passed","time":{"start":1750296245469,"stop":1750297386507,"duration":1141038}},{"uid":"a5beb9b30c9d99e4","reportUrl":"../1812//#testresult/a5beb9b30c9d99e4","status":"passed","time":{"start":1750209872540,"stop":1750210911563,"duration":1039023}},{"uid":"a2a58a69d3d3b6cd","reportUrl":"../1807//#testresult/a2a58a69d3d3b6cd","status":"passed","time":{"start":1750123438661,"stop":1750124527143,"duration":1088482}},{"uid":"4faeb45d75ba4415","reportUrl":"../1806//#testresult/4faeb45d75ba4415","status":"broken","statusDetails":"mysql.connector.errors.InterfaceError: 2003: Can't connect to MySQL server on '10.163.14.52:3306' (111 Connection refused)","time":{"start":1750037035652,"stop":1750037882565,"duration":846913}},{"uid":"533c71d0d38b8fed","reportUrl":"../1805//#testresult/533c71d0d38b8fed","status":"failed","statusDetails":"AssertionError: assert 'mysql-database' in ['information_schema', 'mysql', 'performance_schema', 'sys']","time":{"start":1749950641629,"stop":1749951560009,"duration":918380}},{"uid":"6f41649c1eeca756","reportUrl":"../1804//#testresult/6f41649c1eeca756","status":"passed","time":{"start":1749864236236,"stop":1749865407159,"duration":1170923}},{"uid":"2d059b7816ccf5f5","reportUrl":"../1803//#testresult/2d059b7816ccf5f5","status":"passed","time":{"start":1749777813537,"stop":1749778692267,"duration":878730}},{"uid":"781491e5dbf1efe8","reportUrl":"../1802//#testresult/781491e5dbf1efe8","status":"passed","time":{"start":1749691501123,"stop":1749692381075,"duration":879952}},{"uid":"d14e5b6640681ece","reportUrl":"../1801//#testresult/d14e5b6640681ece","status":"passed","time":{"start":1749605020858,"stop":1749605978989,"duration":958131}},{"uid":"3a97639488fa1323","reportUrl":"../1797//#testresult/3a97639488fa1323","status":"failed","statusDetails":"AssertionError: assert 'mysql-database' in ['information_schema', 'mysql', 'performance_schema', 'sys']","time":{"start":1749518674724,"stop":1749519561589,"duration":886865}},{"uid":"7223cf4b2e9b04e0","reportUrl":"../1795//#testresult/7223cf4b2e9b04e0","status":"passed","time":{"start":1749432267097,"stop":1749433371328,"duration":1104231}},{"uid":"a73f69feaee215f8","reportUrl":"../1793//#testresult/a73f69feaee215f8","status":"failed","statusDetails":"AssertionError: assert 'mysql-database' in ['information_schema', 'mysql', 'performance_schema', 'sys']","time":{"start":1749345882859,"stop":1749346741627,"duration":858768}},{"uid":"63ae4ce544db654f","reportUrl":"../1791//#testresult/63ae4ce544db654f","status":"passed","time":{"start":1749259477425,"stop":1749260535019,"duration":1057594}},{"uid":"4e8d9e4d651fc27c","reportUrl":"../1789//#testresult/4e8d9e4d651fc27c","status":"passed","time":{"start":1749173028949,"stop":1749174119315,"duration":1090366}},{"uid":"6ba3956e35dcfda6","reportUrl":"../1787//#testresult/6ba3956e35dcfda6","status":"broken","statusDetails":"juju.errors.JujuError: resolving ch:sysbench : resolving with preferred channel: Post \"https://api.charmhub.io/v2/charms/refresh\": http: ContentLength=354 with Body length 0","time":{"start":1749086602319,"stop":1749086627967,"duration":25648}},{"uid":"3d427a5128441ab2","reportUrl":"../1785//#testresult/3d427a5128441ab2","status":"passed","time":{"start":1749000228637,"stop":1749001113026,"duration":884389}},{"uid":"ed0364192e9677d6","reportUrl":"../1782//#testresult/ed0364192e9677d6","status":"failed","statusDetails":"AssertionError: assert 'mysql-database' in ['information_schema', 'mysql', 'performance_schema', 'sys']","time":{"start":1748913827104,"stop":1748914768800,"duration":941696}},{"uid":"c2e8bfb1f44dc827","reportUrl":"../1780//#testresult/c2e8bfb1f44dc827","status":"passed","time":{"start":1748827508936,"stop":1748828603939,"duration":1095003}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"94660e53b6b8f75a.json","parameterValues":[]}